### PR TITLE
Refactor describe blocks in stat_tracker_spec

### PR DESCRIPTION
**Pull request by jacksonvaldez**

### CODE REVIEW

File: spec/stat_tracker_spec.rb [hunk 1]
```diff
@@ -164,29 +164,43 @@
     end
   end
 
-  describe 'Season Stats' do
-    it "#winningest_coach" do
-      expect(stattracker.winningest_coach("20122013")).to eq("Claude Julien")
+  describe 'SeasonStats module methods' do
```
Comment (jaw772) [inline spec/stat_tracker_spec.rb:167]: Nice clean up

**Review by jaw772 — APPROVED** [review]

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -164,29 +164,43 @@
...
     end
 
-    it "#fewest_tackles" do
-      expect(stattracker.fewest_tackles("20122013")).to eq("Sporting Kansas City")
+    describe "#fewest_tackles" do
+      it 'returns the name of the Team with the fewest tackles in the season' do
+        expect(stattracker.fewest_tackles("20122013")).to eq("Sporting Kansas City")
+      end
     end
   end
```
Comment (Henchworm) [inline spec/stat_tracker_spec.rb:204]: Good use of 'end'

**Review by Henchworm — APPROVED** [review]

File: spec/stat_tracker_spec.rb (hunk 1 above)
Comment (tjroeder) [reply spec/stat_tracker_spec.rb:167]: Nice cleanup, much cleaner.

**Review by tjroeder — APPROVED** [review]
Great cleanup and use of additional describe blocks for the methods.

**Pull request merged**